system-variables.md

```diff
@@ -845,14 +846,14 @@ Query OK, 0 rows affected (0.09 sec)
 - Default value: `ON`
 - This variable is used to dynamically control whether the telemetry collection in TiDB is enabled. By setting the value to `OFF`, the telemetry collection is disabled. If the [`enable-telemetry`](/tidb-configuration-file.md#enable-telemetry-new-in-v402) TiDB configuration item is set to `false` on all TiDB instances, the telemetry collection is always disabled and this system variable will not take effect. See [Telemetry](/telemetry.md) for details.
 
-### tidb_enable_tso_follower_proxy <span class="version-mark">New in v5.3</span>
+### tidb_enable_tso_follower_proxy <span class="version-mark">New in v5.3.0</span>
```

Comment:
This causes a few broken anchors in release-5.3.0.md. Could you please fix the broken anchors?
![image](https://user-images.githubusercontent.com/41534398/145509830-2e4b65aa-b538-4cc5-809c-0bfa3754f0b4.png)

Reply:
Fixed :-)